library/src/text/quotes.rs

```diff
@@ -135,17 +141,28 @@ impl<'s> Quotes<'s> {
     /// Norwegian.
     ///
     /// For unknown languages, the English quotes are used.
-    pub fn from_lang(lang: Lang, region: Option<Region>) -> Self {
+    pub fn from_lang(lang: Lang, region: Option<Region>, alternative: bool) -> Self {
         let region = region.as_ref().map(Region::as_str);
+
+        let low_high = ("‚", "‘", "„", "“");
+
         let (single_open, single_close, double_open, double_close) = match lang.as_str() {
-            "de" if matches!(region, Some("CH" | "LI")) => ("‹", "›", "«", "»"),
-            "cs" | "da" | "de" | "et" | "is" | "lt" | "lv" | "sk" | "sl" => {
-                ("‚", "‘", "„", "“")
-            }
-            "fr" => ("‹\u{00A0}", "\u{00A0}›", "«\u{00A0}", "\u{00A0}»"),
+            "de" if matches!(region, Some("CH" | "LI")) => match alternative {
+                false => ("‹", "›", "«", "»"),
+                true => low_high,
+            },
+            "cs" | "da" | "de" | "sk" | "sl" if alternative => ("›", "‹", "»", "«"),
+            "cs" | "da" | "de" | "et" | "is" | "lt" | "lv" | "sk" | "sl" => low_high,
+            "fr" => match alternative {
+                false => ("‹\u{00A0}", "\u{00A0}›", "«\u{00A0}", "\u{00A0}»"),
+                true => ("‘\u{00A0}", "\u{00A0}’", "“\u{00A0}", "\u{00A0}”"),
```

Comment:
```suggestion
                true => ("‘", "’", "“", "”"),
```

We don't have spaces within "english" quotes in french.